dbt-snowflake/src/dbt/adapters/snowflake/catalogs/_built_in.py

```diff
@@ -4,19 +4,21 @@
 from dbt.adapters.catalogs import (
     CatalogIntegration,
     CatalogIntegrationConfig,
+    CatalogRelation,
```

Comment:
Since this is a protocol, you shouldn't need to inherit from it. As long as `BuiltInCatalogRelation` implements the protocol, it should pass type checking.